doc/user-guide.md

```diff
@@ -407,6 +407,10 @@ To implement complex deletion logic, you can add a finalizer to your Custom Reso
 deleted until you remove the finalizer (ie, after your cleanup logic has successfully run). For more information, see the 
 [official Kubernetes documentation on finalizers](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#finalizers).
 
+### Metrics
+
+To learn about how metrics works in operator-sdk read the [metrics section][metrics_doc] of the user documentation.
```

Comment:
Nit: metrics is plural so we drop the s for the plural verb work.
```suggestion
To learn about how metrics work in the Operator SDK read the [metrics section][metrics_doc] of the user documentation.
```
Also unless we're talking about the cmd line tool we refer to it as the Operator SDK in general.